src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
Getters and setters are horrifically bad for readability and maintainability. imo they should never be used, in any project. A property should be a property, not a function.

Since this is not a bug fix, there's no urgency to add this change, and I'd prefer to convert `.node` to a function if we want more behavior than "data property".